tools/bridge_inundation.py

```diff
@@ -0,0 +1,127 @@
+import argparse
+import errno
+import os
+from timeit import default_timer as timer
+
+import geopandas as gpd
+import pandas as pd
+
+
+def bridge_risk_status(hydrofabric_dir: str, flow_file_dir: str, output_dir: str) -> gpd.GeoDataFrame:
+    """
+    This function detect which bridge points are affected by a specified flow file. The function requires a flow file (expected to follow
+    the schema used by 'inundation_mosaic_wrapper') with data organized by 'feature_id' and 'discharge' in cms. The output includes a geopackage
+    containing bridge points labeled as "threatened", "at risk", or "not at risk" based on forcasted discharge compared to preset discharge
+    ("max_discharge" or "max_discharge75").
+
+    Args:
+        hydrofabric_dir (str):    Path to hydrofabric directory where FIM outputs were written by
+                                    fim_pipeline.
+        flow_file_dir (str):      Path to flow file to be used for inundation.
```

Comment:
It's a bit confusing that the variable ends with `_dir` because it is a CSV file.